srcpkgs/siril/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'siril'
+pkgname=siril
+version=0.9.12
+revision=1
+archs="x86_64"
+build_style=gnu-configure
+configure_script=./autogen.sh
+hostmakedepends="pkg-config intltool autoconf automake"
+makedepends="fftw-devel libconfig-devel libopencv-devel libffms2-devel \
```

Comment:
Per our convention, multi-line strings should be indented with a single space on each line, no backslash necessary.

Reply:
Fixed